swarm/runtime/safe_paths.py

```
@@ -46,3 +47,47 @@ def validate_path_component(component: str, name: str = "path component") -> str
+    Args:
+        path: The path string to validate.
+        name: Name of the variable for error messages.
+
+    Returns:
+        The validated path (same as input).
+
+    Raises:
+        ValueError: If path is unsafe.
+    """
+    if not path:
+        raise ValueError(f"{name} cannot be empty")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)